packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -473,6 +473,7 @@ export function buildWebpackBrowser(
 
               for (const action of cacheActions) {
                 fs.copyFileSync(action.src, action.dest, fs.constants.COPYFILE_FICLONE);
+                fs.chmodSync(action.dest, 0o666);
```

Comment:
This is the default mode for most of NodeJs methods, such as `writeFile`

Reply:
I think we should go with `0o644`.  This appears to be the value used for the output of the first build.

Also, can you add a comment explaining that the cache writes entries as readonly so we need to correct the permissions here.  Code Ref: https://github.com/npm/cacache/blob/073fbe1a9f789ba42d9a41de7b8429c93cf61579/lib/util/move-file.js#L36

It also only does it on non-windows systems, not sure whether we should as well.